internal/theme/feature.go

```diff
@@ -0,0 +1,33 @@
+package theme
+
+import (
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/internal/cache"
+)
+
+type FeatureName string
+
+const FeatureNameDeviceIsMobile = FeatureName("deviceIsMobile")
+
+// FeatureTheme defines the method to look up features that we use internally to apply functional
+// differences through a theme override.
+//
+// Since: 2.6
```

Comment:
Is this supposed to be a public API (since it has the Since: line)? If so, this should be in the public theme package rather than internal?

Reply:
No it wasn't - that doc was just over-enthusiastic

Reply:
Fixed